assets/js/index.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 require('hof-theme-govuk');
+
+/* eslint-disable no-undef */
+document.getElementById('add-image-button').classList.remove('visuallyhidden');
+
+document.getElementById('add-image-button').addEventListener('click', function callback() {
```

Comment:
Not a biggie, but if you want to name these function, which will make the stack traces more readable, it would be more helpful to have a unique name that describes what it is they do in context of where/when they are doing it, such as `imgClickCallback`. But not `imgClickCallback`. That's an ugly name.

Reply:
O yes, I remember reading about that or someone telling me such a thing a looooong time ago.  I'll change, good one 👍